lib/ex_statsd.ex

```diff
@@ -25,27 +25,27 @@ defmodule ExStatsD do
   Start the server.
   """
   def start_link(options \\ []) do
```

Comment:
This is really more our oversight than anything else, but if you were able to add typespecs for the options on this function that would be really great.